.github/pull_request_template.md

```diff
@@ -7,8 +7,9 @@ Add something here if you want
 --
 
 ## Issues 🔖
-- Issue 01 -> https://issues/here
-- Issue 02 -> https://issues/here2
+
+- Issue 01 -> <https://issues/here>
+- Issue 02 -> <https://issues/here2>
```

Comment:
This `Closes #` in PR description will automatically close the issue when the PR is merged

```suggestion
- Closes #<https://issues/here>
- Closes #<https://issues/here2>
```